### PR TITLE
ci: Change comment trigger detection method

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

It changes the filter for comment trigger.

## Why is it important?

isCommentTrigger needs a node to works because we use it on a when context without a node provisioned it does not work, thus we changed that function to a triggerBy directive.

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

File: .ci/packaging.groovy [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@fix/isCommentTrigger') _
```
Comment (v1v) [inline .ci/packaging.groovy:3]:
```suggestion
@Library('apm@current') _
```

**Comment by kuisathaverat** [comment]
@v1v /package is a message for the packaging job https://github.com/elastic/beats/blob/7b6c25e30f615e6b16acc780989781fd59f4e219/.ci/packaging.groovy#L27

**Pull request merged**